From 08de9dfdf5485a4480044d9e23b9073175998ad0 Mon Sep 17 00:00:00 2001
From: Matthias Runge <mrunge@redhat.com>
Date: Mon, 6 Oct 2014 12:54:03 +0200
Subject: [PATCH] re-add lesscpy to compile .less

---
 openstack_dashboard/settings.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/openstack_dashboard/settings.py b/openstack_dashboard/settings.py
index 44823e2..2539f6f 100644
--- a/openstack_dashboard/settings.py
+++ b/openstack_dashboard/settings.py
@@ -136,6 +136,7 @@ STATICFILES_FINDERS = (
 )
 
 COMPRESS_PRECOMPILERS = (
+    ('text/less', 'lesscpy {infile}'),
     ('text/scss', 'django_pyscss.compressor.DjangoScssFilter'),
 )
 
