From e8d862e47690c5f1f09966042a7e94fc765f3d98 Mon Sep 17 00:00:00 2001
From: Matthias Runge <mrunge@redhat.com>
Date: Mon, 6 Oct 2014 12:54:03 +0200
Subject: [PATCH] re-add lesscpy to compile .less

---
 openstack_dashboard/settings.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/openstack_dashboard/settings.py b/openstack_dashboard/settings.py
index a3a387d..9c78ec6 100644
--- a/openstack_dashboard/settings.py
+++ b/openstack_dashboard/settings.py
@@ -141,6 +141,7 @@ STATICFILES_FINDERS = (
 )
 
 COMPRESS_PRECOMPILERS = (
+    ('text/less', 'lesscpy {infile}'),
     ('text/scss', 'django_pyscss.compressor.DjangoScssFilter'),
 )
 
