From a2a559abf56d7aca322a98e659131d14c5eac67d Mon Sep 17 00:00:00 2001
From: Matthias Runge <mrunge@redhat.com>
Date: Thu, 13 Mar 2014 12:54:10 +0100
Subject: [PATCH] override base templates

---
 openstack_dashboard/utils/settings.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/openstack_dashboard/utils/settings.py b/openstack_dashboard/utils/settings.py
index 97b1a3e..9929386 100644
--- a/openstack_dashboard/utils/settings.py
+++ b/openstack_dashboard/utils/settings.py
@@ -103,4 +103,4 @@ def update_dashboards(modules, horizon_config, installed_apps):
     horizon_config['panel_customization'] = panel_customization
     horizon_config['dashboards'] = tuple(dashboards)
     horizon_config['exceptions'].update(exceptions)
-    installed_apps.extend(apps)
+    installed_apps[:] = apps + installed_apps
