diff --git a/openstack-dashboard/local/local_settings.py.example b/openstack-dashboard/local/local_settings.py.example
index 104d491..7f01e86 100644
--- a/openstack-dashboard/local/local_settings.py.example
+++ b/openstack-dashboard/local/local_settings.py.example
@@ -9,7 +9,7 @@ LOCAL_PATH = os.path.dirname(os.path.abspath(__file__))
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(LOCAL_PATH, 'dashboard_openstack.sqlite3'),
+        'NAME': '/var/lib/openstack-dashboard/dashboard_openstack.sqlite',
         'TEST_NAME': os.path.join(LOCAL_PATH, 'test.sqlite3'),
     },
 }
