diff -Naur horizon-2012.1.orig/openstack_dashboard/settings.py horizon-2012.1/openstack_dashboard/settings.py
--- horizon-2012.1.orig/openstack_dashboard/settings.py	2012-04-04 02:13:35.000000000 +0000
+++ horizon-2012.1/openstack_dashboard/settings.py	2012-05-31 22:40:08.268637001 +0000
@@ -94,14 +94,12 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'django_nose',
     'horizon',
     'horizon.dashboards.nova',
     'horizon.dashboards.syspanel',
     'horizon.dashboards.settings',
 )
 
-TEST_RUNNER = 'django_nose.NoseTestSuiteRunner'
 AUTHENTICATION_BACKENDS = ('django.contrib.auth.backends.ModelBackend',)
 MESSAGE_STORAGE = 'django.contrib.messages.storage.cookie.CookieStorage'
 
