diff -up openstack_dashboard/settings.py.no_webroot openstack_dashboard/settings.py
--- openstack_dashboard/settings.py.no_webroot	2012-10-17 13:02:49.869147461 +0200
+++ openstack_dashboard/settings.py	2012-10-17 13:03:45.768016867 +0200
@@ -35,12 +35,13 @@ TEMPLATE_DEBUG = DEBUG
 
 SITE_BRANDING = 'OpenStack Dashboard'
 
-LOGIN_URL = '/auth/login/'
-LOGOUT_URL = '/auth/logout/'
+WEBROOT='/dashboard'
+LOGIN_URL = WEBROOT+'/auth/login/'
+LOGOUT_URL = WEBROOT+'/auth/logout/'
 # LOGIN_REDIRECT_URL can be used as an alternative for
 # HORIZON_CONFIG.user_home, if user_home is not set.
 # Do not set it to '/home/', as this will cause circular redirect loop
-LOGIN_REDIRECT_URL = '/dashboard'
+LOGIN_REDIRECT_URL = WEBROOT
 
 MEDIA_ROOT = os.path.abspath(os.path.join(ROOT_PATH, '..', 'media'))
 MEDIA_URL = '/media/'
